sdks/python/apache_beam/io/gcp/bigquery_write_it_test.py

```diff
@@ -545,6 +548,102 @@ def test_big_query_write_temp_table_append_schema_update(self, file_format):
               temp_file_format=file_format))
 
 
+class BigQueryXlangStorageWriteIT(unittest.TestCase):
+  BIGQUERY_DATASET = 'python_xlang_write_'
+
+  ELEMENTS = [
+      # (int, float, string, timestamp, bool, bytes)
+      (
+          1,
+          0.1,
+          'a',
+          Timestamp(seconds=100, micros=10),
+          False,
+          bytes('a', 'utf-8')),
+      (
+          2,
+          0.2,
+          'b',
+          Timestamp(seconds=200, micros=20),
+          True,
+          bytes('b', 'utf-8')),
+      (
+          3,
+          0.3,
+          'c',
+          Timestamp(seconds=300, micros=30),
+          False,
+          bytes('c', 'utf-8')),
+      (
+          4,
+          0.4,
+          'd',
+          Timestamp(seconds=400, micros=40),
+          True,
+          bytes('d', 'utf-8')),
+  ]
+
+  def setUp(self):
+    self.test_pipeline = TestPipeline(is_integration_test=True)
+    self.project = self.test_pipeline.get_option('project')
+
+    self.bigquery_client = BigQueryWrapper()
+    self.dataset_id = '%s%s%d' % (
+        self.BIGQUERY_DATASET, str(int(time.time())), random.randint(0, 10000))
+    self.bigquery_client.get_or_create_dataset(self.project, self.dataset_id)
+    _LOGGER.info(
+        "Created dataset %s in project %s", self.dataset_id, self.project)
+
+    self.expansion_service = ('localhost:%s' % os.environ.get('EXPANSION_PORT'))
```

Comment:
We should be using this expansion service when running the pipeline ?

Also, does this service already get created ?